docs/theme/src/docs.css

```diff
@@ -554,7 +554,7 @@ ul.has-icons, dl.has-icons {
       background: rgba(237, 240, 242, 0.56);
       border-radius: 2px;
       color: #6200EA;
-      padding: 0 8px; }
+      padding: 0 4px; }
```

Comment:
FYI, docs css (for teleport/gravity) resides in the `web` repo. This file is just a build output which gets copied to `/teleport/docs` folder. Which means that all changes to the docs css (both teleport/gravity) should be made in `/web` repo. https://github.com/gravitational/web/tree/master/web-assets/docs